http-client/src/main/java/io/airlift/http/client/Request.java

```diff
@@ -291,4 +293,9 @@ private static URI validateUri(URI uri)
         checkArgument(uri.getPort() != 0, "Cannot make requests to HTTP port 0");
         return uri;
     }
+
+    private static ListMultimap<String, String> createHeadersMultimap()
+    {
+        return Multimaps.newListMultimap(new TreeMap<>(comparing(String::toLowerCase)), ArrayList::new);
```

Comment:
toLowerCase is typically used with a locale

but here you can use `java.lang.String#CASE_INSENSITIVE_ORDER`